### ISSUE TITLE
validation: yadageschemas.load() return value not JSON serializable

### ISSUE THREAD

**Issue by diegodelemos** [diana-hep/yadage-schemas#12]
```python
>>> import yadageschemas
>>> import json
>>> result = yadageschemas.load('workflow.yml', toplevel='github:reanahub/reana-demo-root6-roofit', schema_name='yadage/workflow-schema',schemadir=yadageschemas.schemadir,validate=True)
>>> json.dumps(result)
*** TypeError: Object of type 'dict' is not JSON serializable
```
This shouldn't be the case

**Comment by lukasheinrich** [comment]
closed by `yadage-schemas==0.7.10` through PR https://github.com/diana-hep/yadage-schemas/pull/17 .. added unit test

**Comment by diegodelemos** [comment]
According to https://github.com/reanahub/reana-client/pull/54#issuecomment-358366668 we should upgrade `yadage` versions here and in `reana-workflow-engine-yadage` to ...
```
adage==0.8.3
packtivity==0.10.0
yadage[viz]==0.13.5
yadage-schemas==0.7.16
yadage-httpctrl-server==0.0.2
```
